rxlifecycle/src/main/java/com/trello/rxlifecycle/RxLifecycle.java

```diff
@@ -135,6 +140,59 @@ public Boolean call(R lifecycleEvent) {
         return bind(lifecycle, FRAGMENT_LIFECYCLE);
     }
 
+    /**
+     * Binds the given source a View lifecycle.
+     * <p>
+     * Use with {@link Observable#compose(Observable.Transformer)}:
+     * {@code source.compose(RxLifecycle.bindView(lifecycle)).subscribe()}
+     * <p>
+     * This helper automatically determines (based on the lifecycle sequence itself) when the source
+     * should stop emitting items. For views, this effectively means watching for a detach event and
+     * unsubscribing the sequence when one occurs.
+     * <p>
+     * Note that this will unsubscribe after the first {@link ViewAttachEvent.Kind#DETACH} event is received,
+     * and will not resume if the view is re-attached later.
+     *
+     * @param view the view to bind the source sequence to
+     * @return a reusable {@link Observable.Transformer} that unsubscribes the source during the View lifecycle
+     */
+    public static <T> Observable.Transformer<T, T> bindView(final View view) {
+
+        if (view == null) {
+            throw new IllegalArgumentException("View must be given");
+        }
+
+        return bindView(RxView.detaches(view));
+    }
+
+    /**
+     * Binds the given source a View lifecycle.
+     * <p>
+     * Use with {@link Observable#compose(Observable.Transformer)}:
+     * {@code source.compose(RxLifecycle.bindView(lifecycle)).subscribe()}
+     * <p>
+     * This helper automatically determines (based on the lifecycle sequence itself) when the source
+     * should stop emitting items. For views, this effectively means watching for a detach event and
+     * unsubscribing the sequence when one occurs. Note that this assumes <em>any</em> event
+     * emitted by the given lifecycle indicates a detach event.
+     *
+     * @param lifecycle the lifecycle sequence of a View
+     * @return a reusable {@link Observable.Transformer} that unsubscribes the source during the View lifecycle
+     */
+    public static <T> Observable.Transformer<T, T> bindView(final Observable<Object> lifecycle) {
```

Comment:
@SleeplessByte the issue here arises in that while it does seem like a pointless wildcard, using `Observable<Object>` *doesn't* allow you to pass in things that aren't `Object`, including subclasses.

The string example was just mentioned because if you mix it in a chain with different types, it could be problematic since `detaches` events are all just `Object`. Shouldn't really be a problem though, anyone mixing those chains are probably doing it wrong anyway